packages/core/src/observable/types.ts

```diff
@@ -0,0 +1,26 @@
+/* Observable interop typing. Taken from https://github.com/ReactiveX/rxjs */
```

Comment:
There is no such type in https://github.com/ReactiveX/rxjs/blob/master/src/internal/types.ts

Reply:
Subscribable -> Observable: the rxjs Subscribable is the actual Observable from the TC39 proposal, the rxjs Observable interface has other stuff included
Unsubscribable + { closed: boolean } -> Subscription: 
PartialObserver -> Partial<Observer>: the PartialObserver of rxjs require at least 1 observer method, but this is not required for us and the typing simplifies on our side and its still interoperable

Reply:
Understood. Please remove this comment, since there is nothing taken from rxjs.